src/client/firefox/commands.js

```diff
@@ -317,6 +319,12 @@ async function fetchSources() {
   return sources.map(source => createSource(source, { supportsWasm }));
 }
 
+async function getDeviceDescription() {
+  const listTabsForm = await debuggerClient.listTabs();
+  const deviceFront = await getDeviceFront(debuggerClient, listTabsForm);
```

Comment:
could we add a method to the debuggerClient?

`debuggerClient.getDeviceFront()`

I think we do this in other places too

Reply:
Hmm, do you have an example of another place this is done?

I imagine the `debuggerClient` as for "interacting with the root actor", so `getDeviceFront` is not really related to that in my mental model.

Reply:
Hmm, was thinking of cases where clients isntantiate other clients or manage relation them:
- https://searchfox.org/mozilla-central/source/devtools/shared/client/thread-client.js#493
- https://searchfox.org/mozilla-central/source/devtools/shared/client/tab-client.js#67-70

Reply:
Ah, I see... These are all examples from older, pre-protocol.js actors, whereas device actor uses protocol.js. We don't currently have an obvious API to jump between old style actor parent to new style actor child like this. At the moment, it's typically done manually in a manner similar to this PR:

https://searchfox.org/mozilla-central/search?q=new.*Front%5C(&case=true&regexp=true&path=

Seems like it would be worth exploring a natural way to do this for all actors in a follow up bug.